samples/apps/copilot-chat-app/webapp/src/App.tsx

```diff
@@ -69,16 +69,29 @@ const App: FC = () => {
     const dispatch = useAppDispatch();
 
     const { instance, inProgress } = useMsal();
-    const account = instance.getActiveAccount();
+    const { activeUserInfo } = useAppSelector((state: RootState) => state.app);
     const isAuthenticated = useIsAuthenticated();
 
     const chat = useChat();
 
     useEffect(() => {
-        if (isAuthenticated && account) {
-            dispatch(setLoggedInUserId(account.homeAccountId));
+        if (isAuthenticated) {
+            let isActiveUserInfoSet = activeUserInfo !== undefined;
```

Comment:
should we add `activeUserInfo` to the dependency array for the hook?

Reply:
I don't think that's necessary because isActiveUserInfoSet is set to true, so the next if block should run / chats should be loaded as soon as the activeUser is set in app state.

Reply:
it's good practice to add it unless we have a specific reason to exclude it. I can push this change here in a sec and will make sure it doesn't cause any unnecessary renders.
ideally, we can move away from `// eslint-disable-next-line react-hooks/exhaustive-deps` lines except for in rare cases where unavoidable.

Reply:
@dehoward I tested it and it seems like it introduces more calls to the token cache? Can you test it on a production build and see if you're able to repro?
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/2fbab4fc-7145-4910-8aaf-0c9c16ff2b98)

Reply:
I just noticed this too, looks like there are a couple of more calls. let's not block the PR for this. 

I will go through soon and look to remove all of the `eslint-disable-next-line` lines and make sure we don't add any new calls, as usually it will reduce them when done correctly.